test/e2e_node/system/kernel_validator.go

```diff
@@ -78,7 +78,7 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.Infof("Validating kernel version")
+	fmt.Printf("Validating kernel version")
```

Comment:
I wonder if this is worth changing since it's in another part of the system.

Reply:
If we don't keep thius change then we get a glog in the middle of our nice output. If we do keep this change, it's changing another part of the system and they probably want glog output.

Reply:
IMO we should consider also the option to remove this fmt/glog,

Reply:
@fabianofranz I think you're suggesting I change this to `glog.V(1).Infof` which gives us the option to turn it off and on? If you're talking about something else could you elaborate?

Reply:
I think I'm going to nix this change and we'll figure out a better solution / I'll go talk to the testing folks and see what they think.

Reply:
@chuckha sorry if I wasn't clear enough.
If the log entry `Validating kernel version` is not  functional to something, we can consider to remove line 81 instead of either keeping glog (annoying for kubeadm output) or using fmt (that might be annoying for e2e test)
However for both options we should talk to testing folks, so ok with the approach you are proposing